tests/integration/regression_test.py

```diff
@@ -14,7 +14,7 @@ def test_443_handle_nonchunked_response_in_stream(self):
         with pytest.raises(docker.errors.APIError) as exc:
             for line in self.client.build(fileobj=dfile, tag="a/b/c"):
                 pass
-        assert exc.value.response.status_code == 500
+        assert exc.value.response.is_error()
```

Comment:
Removing `.response` should clear up the CI failure

Reply:
🤦‍♂️ updated again; thanks for bearing with me 🤗